app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.add_instance()
+        self.people[name] = self
+
+    def add_instance(self) -> None:
+        Person.people[self.name] = self
```

Comment:
```suggestion
        self[name] = self
```

Reply:
please, if you do not understand, please sign up at Q&A, or ask a question in the Slack chat